renderer/views/player.js

```diff
@@ -116,6 +117,7 @@ function renderPlayerControls (state, dispatch) {
 
   // Handles a click or drag to scrub (jump to another position in the video)
   function handleScrub (e) {
+    dispatch('videoMouseMoved')
```

Comment:
do we need this? if we're in `handleScrub`, we just had a mousemove event anyway, right?